src/test/java/org/folio/rest/impl/PostGobiOrdersHelperTest.java

```diff
@@ -7,12 +7,17 @@
 import static org.junit.Assert.fail;
 
 import java.io.StringReader;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.CompletionException;
 
 import javax.xml.bind.JAXBContext;
 import javax.xml.bind.JAXBException;
 import javax.xml.bind.Unmarshaller;
 
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import org.apache.commons.collections4.map.HashedMap;
```

Comment:
Is this import used? The only reason I ask is that unless HashedMap provides something above and beyond HashMap or is required for a 3rd party library, I think it would be best to stick with built in classes. If it isn't used, let's remove it so my eclipse won't show me a warning.

Reply:
They were used for a prior test that I deleted.  I just checked in changes to address your comments.